packages/eslint-plugin/src/rules/prefer-nullish-coalescing.ts

```diff
@@ -7,10 +7,16 @@ import * as util from '../util';
 
 export type Options = [
   {
+    allowRuleToRunWithoutStrictNullChecksIKnowWhatIAmDoing?: boolean;
     ignoreConditionalTests?: boolean;
-    ignoreTernaryTests?: boolean;
     ignoreMixedLogicalExpressions?: boolean;
-    allowRuleToRunWithoutStrictNullChecksIKnowWhatIAmDoing?: boolean;
+    ignorePrimitives?: {
+      bigint?: boolean;
+      boolean?: boolean;
+      number?: boolean;
+      string?: boolean;
+    };
```

Comment:
[Non-actionable] In theory we could allow this to be a `boolean` to configure all of the primitives at once. I think it's fine to leave that as a follow-up to this PR.

Reply:
Good idea, leaving it for now.

Reply:
Do I need to open an issue for it or can I reference the same one?

Reply:
We should have a new issue for tracking. It gets confusing having multiple PRs target the same issue.

Reply:
#7180 ✨